Backend/utils/user.utils.js

```diff
@@ -35,6 +35,20 @@ function generateToken(username) {
   return accesstoken;
 }
 
+function removeToken(res){
+
+    res.status(200)
+    .cookie('jwt', null, {
+        expiresIn: new Date(Date.now())
+    }, 
+    {httpOnly: true})
+    .json({
+        success: true,
+        message: "Logged Out"
+    });
+
+}
```

Comment:
Deleting the cookie should be enough. refer to this link for better understanding, https://stackoverflow.com/questions/37959945/how-to-destroy-jwt-tokens-on-logout